cli/cmd/plugin/unmanaged-cluster/cluster/minikube.go

```diff
@@ -0,0 +1,310 @@
+// Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cluster
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/vmware-tanzu/community-edition/cli/cmd/plugin/unmanaged-cluster/config"
+
+	"k8s.io/klog/v2"
+	minikubeConfig "k8s.io/minikube/pkg/minikube/config"
+)
+
+var minikubeBinName = "minikube"
+
+type MinikubeClusterManager struct{
+	// TODO - What other minikube specific options should we implement?
+	driver string
```

Comment:
What other minikube specific options should we provide? Many of the flags on `start` are driver specific too.

One thought I had was adding a 
```
ProviderConfiguration:
  rawMinikubeArgs: ...
```
so someone could drop in their driver specific arguments since there are so many possible edge cases with minikube

Reply:
yea, seems like that could be a good idea otherwise you might need to do a "helm" and expose every option

Reply:
Seems like a fine solution.

Most users can just pre-configure minikube too.

```
minikube config set driver kvm2
```

Reply:
@jpmcb I wouldn't mind seeing `container-runtime` at the root level too. Otherwise, everything can go in raw.